### PR TITLE
chore(twig) - setup cypress tests 

### PR DESCRIPTION
**Pull request by Shashika6**
Fixes :- https://github.com/international-labour-organization/designsystem/issues/1117

Notes :- 

* Migrate the cypres tests and run it against ilo_base_theme

- [x] Add cypress to new twig 
- [x] Add cypress tests to new twig 
- [x] Add docker container to run theme locally.
- [x] Add cypress scripts to package.json

### CODE REVIEW

**Comment by Shashika6** [comment]
@bashlk will you be able to review this one :) before merging ?

File: packages/twig-new/cypress/e2e/content/richtext.cy.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//WIP
```
Comment (bashlk) [inline packages/twig-new/cypress/e2e/content/richtext.cy.js:1]: Another WIP

File: packages/twig-new/cypress/e2e/feedback/modal.cy.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+//WIP
```
Comment (bashlk) [inline packages/twig-new/cypress/e2e/feedback/modal.cy.js:1]: Again

File: packages/twig-new/cypress/e2e/feedback/notification.cy.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//WIP
```
Comment (bashlk) [inline packages/twig-new/cypress/e2e/feedback/notification.cy.js:1]: And again

File: packages/twig-new/cypress/e2e/transitions/loading.cy.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+//WIP
```
Comment (bashlk) [inline packages/twig-new/cypress/e2e/transitions/loading.cy.js:1]: More

File: packages/twig-new/src/components/tags/tag.component.yml [hunk 5]
```diff
@@ -8,20 +8,17 @@ tag:
       type: text
       label: Content
       description: The tag text content.
-      preview:
-        faker: lorem.word
+      preview: content
```
Comment (bashlk) [inline packages/twig-new/src/components/tags/tag.component.yml:11]: This is value that is supplied as the default to the field right? Maybe Tag is better text for here

File: packages/twig-new/src/components/tags/tag.component.yml [hunk 6]
```diff
@@ -8,20 +8,17 @@ tag:
...
     elementId:
       type: text
       label: ID
       description: The tag's id.
-      preview:
-        faker: lorem.word
+      preview: contentId
```
Comment (bashlk) [inline packages/twig-new/src/components/tags/tag.component.yml:16]: This is elementId

**Review by bashlk — CHANGES_REQUESTED** [review]

File: packages/twig-new/cypress/e2e/content/richtext.cy.js (hunk 1 above)
Comment (Shashika6) [reply packages/twig-new/cypress/e2e/content/richtext.cy.js]: Added a better description on this

File: packages/twig-new/cypress/e2e/feedback/modal.cy.js (hunk 2 above)
Comment (Shashika6) [reply packages/twig-new/cypress/e2e/feedback/modal.cy.js]: Added a better description on this

File: packages/twig-new/cypress/e2e/feedback/notification.cy.js (hunk 3 above)
Comment (Shashika6) [reply packages/twig-new/cypress/e2e/feedback/notification.cy.js]: Added a better description on this

**Pull request merged**